packages/oats-runtime/test/shape-of.spec.ts

```diff
@@ -82,9 +82,13 @@ describe('ShapeOf', () => {
   });
 
   it('works with nested branded types', () => {
+    // @ts-expect-error -- does not work with nested branded types
```

Comment:
erm. so updating ts breaks ShapeOf? any idea why

Reply:
No idea. This works however:


```
type NestedValue = runtime.BrandedScalar<'a' | 'b' | 'c' | 'd', BrandOfNestedValue>;
```

Reply:
The update doesn't change the produced output at least for me. So, this PR is a no-op in that sense and only makes it easier to maybe address this and other issues.